### PR TITLE
Docs explaining the new ingress-to-route controller

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
Docs associated with Claytons addition of a route-to-ingress controller
[PR18658](https://github.com/openshift/origin/pull/18658)

### CODE REVIEW

**Comment by JacobTanenbaum** [comment]
@knobunc PTAL

File: architecture/networking/routes.adoc [hunk 1]
```diff
@@ -833,3 +833,53 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
 $ oc expose service/<name> --hostname="kates.net"
 $ oc expose service/<name> --hostname="api.kates.net"
 ----
+
+
+== Support for Kubernetes Ingress objects in Openshift
+
+In an effort to support the Kubernetes method of managing external access to services
+in a cluster an ingress to route controller has been created. This controller watches
```
Comment (knobunc) [inline architecture/networking/routes.adoc]: Can we use Ingress and Route consistently (or ingress and route)

**Review by knobunc — CHANGES_REQUESTED** [review]
We should mention how this works with claiming rules, and how secrets are handled.

Also we should talk about when the objects are deleted.

File: architecture/networking/routes.adoc [hunk 2]
```diff
@@ -833,3 +833,53 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
 $ oc expose service/<name> --hostname="kates.net"
 $ oc expose service/<name> --hostname="api.kates.net"
 ----
+
+
+== Support for Kubernetes Ingress objects in Openshift
```
Comment (Miciah) [inline architecture/networking/routes.adoc]: "OpenShift"

File: architecture/networking/routes.adoc [hunk 3]
```diff
@@ -833,3 +833,53 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
...
+           servicePort: 80
+----
+
+would generate
+
+----
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: test-a34th
```
Comment (Miciah) [inline architecture/networking/routes.adoc]: Maybe a callout to say that the name is automatically generated, with the ingress's name as a prefix.

Comment (JacobTanenbaum) [reply architecture/networking/routes.adoc]: Added in the description

File: architecture/networking/routes.adoc [hunk 4]
```diff
@@ -833,3 +833,59 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
...
+
+----
+kind: Ingress
+apiVersion: extensions/v1beta1
+metadata:
+  name: test
+spec:
+  rules:
+  - host: test.com
+     http:
```
Comment (zhaozhanqi) [inline architecture/networking/routes.adoc]: there is extra spaces before `http:`

**Comment by bfallonf** [comment]
Verified with @JacobTanenbaum that this was technically ok. I made a new PR for this:

https://github.com/openshift/openshift-docs/pull/8638

@zhaozhanqi this included validating the YAML. Please let me know if there's anything more needed.

**Pull request closed without merging**